### ISSUE TITLE
Doesn't display always

### ISSUE THREAD

**Issue by tchesa** [webBoxio/atom-color#39]
Doesn't display when just open the css file. When you scroll off the color line and come back, the color display disappear. Just display again when you change the color line or a line above.

**Comment by eved42** [comment]
I have the same problem. When I uncheck "Fill color as background" to see color overview next to color code, I see the color when I open the file and it disappears when I scroll the file.

And besides, the overview doesn't display for every codes in a less file : 

<a href="http://www.hostingpics.net/viewer.php?id=95928320160930102057contentlessCwamp64wwwsymfony30localproductsAtom.png"><img src="http://img4.hostingpics.net/thumbs/mini_95928320160930102057contentlessCwamp64wwwsymfony30localproductsAtom.png" alt="Hébergement photos" /></a>

**Comment by josuedanr** [comment]
yeah same for me i thought that unchecking the "Fill color as background" option would fix it but it still doesnt work.

![image](https://cloud.githubusercontent.com/assets/15602952/19412505/4c133c32-92d5-11e6-90fb-44a3555da9dc.png)

when i rewrite a the color it seems to work 

![image](https://cloud.githubusercontent.com/assets/15602952/19412528/b16d9c8a-92d5-11e6-9f1a-d6979f0e044b.png)

but if you scroll down or up it disappears again.

**Comment by emilianos** [comment]
same here, if you scroll they all go away.
the custom styling is not working for me eigther 
and it's really messing with the line heights it seems..

![screen shot 2016-10-19 at 14 25 30](https://cloud.githubusercontent.com/assets/4334689/19530033/828f3bcc-9608-11e6-9a60-937ef9c1d2cc.png)

![screen shot 2016-10-19 at 14 30 16](https://cloud.githubusercontent.com/assets/4334689/19530055/9ab71a94-9608-11e6-9119-f74207e3e5b0.png)

it would also be good to have support for variables

this is a similar sublime extension that features that:
https://github.com/Monnoroch/ColorHighlighter

**Comment by emilianos** [comment]
well, the styles work if placed inside `atom-text-editor, atom-text-editor::shadow {`
with that one can force width and height so the line problem is gone.
```
height: 15px !important;
width: 15px !important;
```

**Comment by eved42** [comment]
I changed for [Pigments package](https://atom.io/packages/pigments) which works perfectly.

**Comment by emilianos** [comment]
nice find! thanks for sharing @eved42
